datalab/datalab_session/util.py

```diff
@@ -95,11 +97,11 @@ def get_archive_from_basename(basename: str) -> dict:
 
   response = requests.get(settings.ARCHIVE_API + '/frames/', params=query_params, headers=headers)
 
-  try:
-    image_data = response.json()
-    results = image_data.get('results', None)
-  except Exception as e:
-    raise FileNotFoundError(f"Error fetching image data from archive: {e}")
+  image_data = response.json()
+  results = image_data.get('results', None)
```

Comment:
you should also do a response.raise_for_status() and wrap it in try/except before this because getting a 500 or something will result in no json being there so response.json() will crash.